### ISSUE TITLE
use whittaker-shannon interpolation when going from  default grid to Nyquist-Shannon

### ISSUE THREAD

**Issue by sbillinge** [diffpy/diffpy.pdfgui#112]

**Comment by bobleesj** [comment]
Not sure if want to include this for v3.1.0? @sbillinge

**Comment by sbillinge** [comment]
no, lets push that to the next release.  I pulled it off the milestons

**Comment by sbillinge** [comment]
Let's bump this back up now we have nsinterp merged in utils.

It is a one line fix that isn't visible to users so I suggest we do a hot fix and include it in the upcoming non-rc 3.1.0 release.

@Sparks29032 can do it, or @bobleesj if you like.
